### PR TITLE
Cleanup postgresql usage.

### PR DESCRIPTION
**Pull request by vjkoskela**
Remove the generic query generator and h2 migrations. Simply the code and tests as possible.

### CODE REVIEW

File: app/com/arpnetworking/metrics/portal/alerts/impl/DatabaseAlertRepository.java [hunk 1]
```diff
@@ -62,34 +61,22 @@
      * @param environment Play's <code>Environment</code> instance.
      * @param config Play's <code>Configuration</code> instance.
      * @param ebeanServer Play's <code>EbeanServer</code> for this repository.
-     * @throws Exception If the configuration is invalid.
      */
     @Inject
     public DatabaseAlertRepository(
             final Environment environment,
```
Comment (BrandonArp) [inline app/com/arpnetworking/metrics/portal/alerts/impl/DatabaseAlertRepository.java]: Can you remove the Environment and Config parameters?

File: app/com/arpnetworking/metrics/portal/alerts/impl/DatabaseAlertRepository.java [hunk 2]
```diff
@@ -62,34 +61,22 @@
...
-     * Public constructor.
+     * Public constructor for manual configuration. This is intended for testing.
      *
      * @param ebeanServer Play's <code>EbeanServer</code> for this repository.
-     * @param alertQueryGenerator Instance of <code>AlertQueryGenerator</code>.
      */
-    public DatabaseAlertRepository(
-            final EbeanServer ebeanServer,
-            final AlertQueryGenerator alertQueryGenerator) {
+    public DatabaseAlertRepository(final EbeanServer ebeanServer) {
```
Comment (BrandonArp) [inline app/com/arpnetworking/metrics/portal/alerts/impl/DatabaseAlertRepository.java]: Can you please unify the 2 constructors?

File: app/com/arpnetworking/metrics/portal/hosts/impl/DatabaseHostRepository.java [hunk 3]
```diff
@@ -64,34 +63,22 @@
...
-     * Public constructor.
+     * Public constructor for manual configuration. This is intended for testing.
      *
      * @param ebeanServer Play's <code>EbeanServer</code> for this repository.
-     * @param hostQueryGenerator Instance of <code>HostQueryGenerator</code>.
      */
-    public DatabaseHostRepository(
-            final EbeanServer ebeanServer,
-            final HostQueryGenerator hostQueryGenerator) {
+    public DatabaseHostRepository(final EbeanServer ebeanServer) {
```
Comment (BrandonArp) [inline app/com/arpnetworking/metrics/portal/hosts/impl/DatabaseHostRepository.java]: Please unify constructors

File: app/com/arpnetworking/metrics/portal/reports/impl/DatabaseReportRepository.java [hunk 4]
```diff
@@ -92,42 +90,29 @@
...
-     * Public constructor.
+     * Public constructor for manual configuration. This is intended for testing.
      *
      * @param ebeanServer Play's <code>EbeanServer</code> for this repository.
-     * @param reportQueryGenerator Instance of <code>ReportQueryGenerator</code>.
      */
-    public DatabaseReportRepository(
-            final EbeanServer ebeanServer,
-            final ReportQueryGenerator reportQueryGenerator) {
+    public DatabaseReportRepository(final EbeanServer ebeanServer) {
```
Comment (BrandonArp) [inline app/com/arpnetworking/metrics/portal/reports/impl/DatabaseReportRepository.java]: Unify constructors

**Comment by vjkoskela** [comment]
Discussed the reason for having the two constructors; one is for loading repository dependencies from configuration injected by the running application and the other is for providing the same dependencies from tests. 

While there is room for improvement regarding use of sub-configurations, the direct reading of configuration in classes and so forth. These would remove the duplicate constructors, but probably not providing enough value to justify doing it at the moment.

**Pull request merged**